docs/sdk/web3-react/guides/01-connect-wallet.md

```diff
@@ -1,4 +1,165 @@
 ---
 id: connect-wallet
 title: Connecting to Wallets
----     
\ No newline at end of file
+---     
+
+## Introduction
+
+This guide will cover how to connect wallets with `web3-react`. It is based on the [web3-react example](https://github.com/Uniswap/examples/tree/main/web3-react), found in the Uniswap code examples [repository](https://github.com/Uniswap/examples). To run this example, check out the examples's [README](https://github.com/Uniswap/examples/blob/main/web3-react/README.md) and follow the setup instructions.
+
+
+In this example we will walk through setting up `web3-react` and connecting the most popular browser injected connector, *MetaMask*, using the web3-react [MetaMask package](https://www.npmjs.com/package/@web3-react/metamask).
+
+
+The input parameters to this guide are the chains that we want our app to be able to connect to and their RPC URLs.
+
+The guide will **cover**:
+
+1. Creating a `Web3ReactProvider`
+2. Building an `InjectedConnector`
+3. Connecting and  disconnecting the application to the connector
+
+At the end of the guide, we should be able to connect and disconnect the application to a MetaMask injected connector.
+
+For this guide, the following `web3-react` packages are used:
+
+- [`@web3-react/core`](https://www.npmjs.com/package/@web3-react/core)
```

Comment:
It's strange that the npm packages are called `@web3-react/*`, but you refer to them as web3-react and web3-react MetaMask package, etc. I think you should always call them by their package name to avoid confusion.

Reply:
Changed!